src/pybind/mgr/dashboard/frontend/cypress/integration/page-helper.po.ts

```diff
@@ -260,7 +261,7 @@ export abstract class PageHelper {
   clearTableSearchInput() {
     this.waitDataTableToLoad();
 
-    return cy.get('cd-table .search button').click();
+    return cy.get('cd-table .search button').click({ multiple: true });
```

Comment:
Can we consider another alternatives to `multiple: true` option?
This is like clicking n multiple items.
It would be nicer to improve the CSS selector or going for the appropriate item (e.g. first()), like:
cy.get('cd-table .search button').first().click();

Reply:
Thanks, Updated as per your suggestion.